setup-signify.sh

```diff
@@ -1,13 +1,21 @@
 #!/bin/bash
 set -euo pipefail 
 
-sudo apt -y install signify-openbsd qrencode
+if [ $# -eq 1 ]; then
+    wd=$1
```

Comment:
```suggestion
  public_path="${1}/key.pub"
  secret_path="${1}/key.sec"
```

Then use `"${public_path}"` and `"${secret_path}"` (with double-quotes) below.